components/faq/question.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './question.css';
+
+Question.propTypes = {
+  question: PropTypes.string.isRequired,
+  answer: PropTypes.node.isRequired,
+};
+
+function Question({ question, answer }) {
```

Comment:
I know this was just copy/paste for ya, so I converted the function from ES6 --> Vanilla 👍